components/devtools/actors/browsing_context.rs

```diff
@@ -229,7 +229,10 @@ impl BrowsingContextActor {
 
         let accessibility = AccessibilityActor::new(actors.new_name("accessibility"));
 
-        let css_properties = CssPropertiesActor::new(actors.new_name("css-properties"));
+        let (tx, rx) = ipc::channel().unwrap();
+        script_sender.send(GetCssDatabase(tx)).unwrap();
+        let properties = rx.recv().unwrap();
```

Comment:
should we at least use `expect` with message or print warning for `.send` and `.recv`?

Reply:
Definitely, I was quickly testing that everything worked but it needs some clean up. Thank you for pointing it out!